docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,380 @@
+            stdy = np.abs(std_pos*np.cos(el_az_range[1]))
+            stdz = np.abs(detection.state_vector[2, 0]*el_az_range[0])
+            if stdx > MAX_DEV:
+                print('Warning - X Deviation exceeds limit!!')
+            if stdy > MAX_DEV:
+                print('Warning - Y Deviation exceeds limit!!')
+            if stdz > MAX_DEV:
+                print('Warning - Z Deviation exceeds limit!!')
+            C0 = np.diag(np.array([stdx, 50.0, stdy, 50.0, stdz, 10.0])**2)
+
+            tracks.add(Track([GaussianStateUpdate(
+                state_vector,
+                C0,
+                SingleHypothesis(None, detection),
+                timestamp=detection.timestamp)
+            ]))
+        return tracks
+
+
+meas_model = CartesianToElevationBearingRange(
+            ndim_state=6,
+            mapping=np.array([0, 2, 4]),
+            noise_covar=noise_covar)
+
+prior_state = GaussianState(
+        np.array([[0], [0], [0], [0], [0], [0]]),
+        np.diag([1000, 50, 1000, 50, 1000, 10.0])**2)
+
+initiator = Initiator(prior_state, measurement_model=meas_model)
+
+# %%
+# Now that we have setup out tracking scenario we can wrap our simulation environment within a
+# :class:`~.MultiTargetTracker`. This takes our tracker configurations and the simulation we previously created and
+# brings them together into a single iterable object.
+
+from stonesoup.hypothesiser.distance import DistanceHypothesiser
+from stonesoup.measures import Mahalanobis
+hypothesiser = DistanceHypothesiser(predictor, updater, measure=Mahalanobis(), missed_distance=3)
+
+from stonesoup.dataassociator.neighbour import NearestNeighbour
+data_associator = NearestNeighbour(hypothesiser)
+
+deleter = UpdateTimeStepsDeleter(time_steps_since_update=2)
+
+# Create a Kalman multi-target tracker
+kalman_tracker = MultiTargetTracker(
+    initiator=initiator,
+    deleter=deleter,
+    detector=sim,
+    data_associator=data_associator,
+    updater=updater
+)
+
+# %%
+# The final step is to iterate our tracker over the simulation:
+kalman_tracks = {}  # Store for plotting later
+groundtruth_paths = {}  # Store for plotting later
+detections = []  # Store for plotting later
+
+for time, ctracks in kalman_tracker.tracks_gen():
+    for track in ctracks:
+        loc = (track.state_vector[0], track.state_vector[2])
+        if track not in kalman_tracks:
+            kalman_tracks[track] = []
+        kalman_tracks[track].append(loc)
+
+    for truth in groundtruth_sim.current[1]:
+        loc = (truth.state_vector[0], truth.state_vector[2])
+        if truth not in groundtruth_paths:
+            groundtruth_paths[truth] = []
+        groundtruth_paths[truth].append(loc)
+
+    for detection in sim.detections:
+        detect_state = detection.measurement_model.inverse_function(detection)
+        loc = (detect_state[0], detect_state[2])
+        detections.append(loc)
+
+# %%
+# Plotting the tracker output we get:
```

Comment:
Give us a key or describe what's what in the plot. I can't tell which line is which.